util/generate-rsa-signing-tests.py

```diff
@@ -26,10 +26,26 @@
 from cryptography.hazmat.primitives.asymmetric import rsa, padding
 import sys, copy
 
-def parse(fn):
+DIGEST_OUTPUT_LENGTHS = {
+    'SHA1': 80,
+    'SHA224': 112,
```

Comment:
'SHA224` should be removed since it's not a supported algorithm and it is never used, IIUC.